readme.md

```diff
@@ -1,6 +1,6 @@
 # IPFS CID Inspector
 
-<img width="1117" alt="screenshot 2018-02-21 11 47 04" src="https://user-images.githubusercontent.com/58871/36478523-fd27fc7c-16fc-11e8-8790-310719700de1.png">
+<img width="1117" alt="screenshot 2018-02-21 12 06 38" src="https://user-images.githubusercontent.com/58871/36479196-b6da0a28-16ff-11e8-9949-d8cb72343367.png">
```

Comment:
Also, probably we should put this image on IPFS or in the Git repository, not sure how long time Github keeps comment images around.

Reply:
I've been (ab)using github as a image host via comments for a good few years now without problems, but happy to add the image to the repo for ipfs-ing.